docs/source/federated-types/federated-directives.mdx

```diff
@@ -911,8 +911,26 @@ directive @context(name: String!) on OBJECT | INTERFACE | UNION;
 
 A `@context` directive must be applied to an object, interface, or union type. A type can be annotated with one or more `@context` directives.
 
-Each `@context` must be defined with a name that's unique within the supergraph.
+Each `@context` must be defined with a name that's unique within the subgraph. 
 
+Each `@context` can be applied to multiple places within a subgraph. For example:
+
+```graphql
+type A @key(fields: "id") @context(name: "ctxt") {
```

Comment:
Cspell doesn't like the name ctxt (which I know I suggested). Either change it to a real word or add it to cspell-dict.txt